File: FLUTTER/igest_app/lib/models/principal/perfil.dart
class Perfil {
  int idPerfil;
  String idSistema;
  String nomPerfil;
  bool activo;
  int idTipoEmpresa;
  String descripcion;
  int idEmpresa;

  Perfil({
    required this.idPerfil,
    required this.idSistema,
    required this.nomPerfil,
    required this.activo,
    required this.idTipoEmpresa,
    required this.descripcion,
    required this.idEmpresa,
  });

  factory Perfil.fromJson(Map<String, dynamic> json) {
    return Perfil(
      idPerfil: json['id_perfil'],
      idSistema: json['id_sistema'],
      nomPerfil: json['nom_perfil'],
      activo: json['activo'],
      idTipoEmpresa: json['id_tipo_empresa'],
      descripcion: json['descripcion'],
      idEmpresa: json['id_empresa'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id_perfil': idPerfil,
      'id_sistema': idSistema,
      'nom_perfil': nomPerfil,
      'activo': activo,
      'id_tipo_empresa': idTipoEmpresa,
      'descripcion': descripcion,
      'id_empresa': idEmpresa,
    };
  }
}

File: NET/_XXX_com.yupanastudio.polariss.app401.model/admin/Perfil.cs
using System.Collections.Generic;

namespace com.yupanastudio.polariss.app
{
    public class Perfil
    {
        public int idPerfil { get; set; }
        public string idSistema { get; set; }
        public string nomPerfil { get; set; }
        public bool activo { get; set; }
        public int idTipoEmpresa { get; set; }
        public string descripcion { get; set; }
        public int idEmpresa { get; set; }
        public int idUsuPerfilX { get; set; }
        public long idUsuarioX { get; set; }
        public bool activoX { get; set; }
        //public bool selX { get; set; }
        public bool nuevoX { get; set; }
        public bool modificarX { get; set; }
        public bool eliminarX { get; set; }
        public bool imprimirX { get; set; }
        public List<Opcion> lsOpcionX { get; set; }

    }//end class

}
